exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/traces/ProtoSpanFlagsUtil.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal.otlp.traces;
+
+import io.opentelemetry.api.trace.SpanContext;
+
+public final class ProtoSpanFlagsUtil {
+
+  private ProtoSpanFlagsUtil() {}
+
+  public static final int SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK = 0x00000100;
+  public static final int SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK = 0x00000200;
+
+  static int getFullSpanFlags(byte w3cTraceflags, SpanContext parentContext) {
```

Comment:
This is pretty specific to the remote flags. What about a name like `setRemoteContextFlags()` instead?